Documentation/FrameworkOverview.md

```diff
@@ -136,15 +136,15 @@ For interaction with `NSControl` or `UIControl`, RAC provides the
 
 ## Properties
 
-A **property**, represented by the [`PropertyType`][Property] protocol,
+A **property**, represented by the [`PropertyProtocol`][Property],
 stores a value and notifies observers about future changes to that value.
 
 The current value of a property can be obtained from the `value` getter. The
 `producer` getter returns a [signal producer](#signal-producers) that will send
-the property’s current value, followed by all changes over time.
+the property’s current value, followed by all changes over time. The `signal` getter returns a [signal](#signals) that will send all changes over time, but not the initial value.
 
 The `<~` operator can be used to bind properties in different ways. Note that in
-all cases, the target has to be a [`MutablePropertyType`][Property].
+all cases, the target has to be a [`BindingTarget`][BindingTarget], [`MutableProperty`][MutableProperty] is the only property that implements this.
```

Comment:
`BindingTarget` is a concrete type. Maybe 

> binding target, represented by `BindingTargetProtocol`

instead?

Moreover, `MutableProperty` is not the only property that can be a target, since you can have any arbitrary number of mutable property types. I'd prefer to say:

> All mutable property types are inherently a binding target.